src/itdelatrisu/opsu/options/Options.java

```diff
@@ -196,7 +196,7 @@ private static File getXDGBaseDir(String env, String fallback) {
 	}
 
 	/**
-	 * Returns the osu! installation directory.
+	 * Returns the osu! installation directory (currently supports Windows only)
```

Comment:
> osu! only runs on Windows anyway...

Wine and PlayOnLinux allow osu! to be _run_ from Linux machines, although this is considered a "hack" because it was never intended to be _played_ on Linux. (Although the new client allows it to be compiled and run on Linux, but its status is still under Alpha and its really incomplete as of this PR.)

And I think what you meant to say was:
> osu! only runs **natively** on Windows anyway...

**Removed additional comment** as per request